src/objects/exchangeable_obj.f90

```diff
@@ -157,7 +158,79 @@ module subroutine retrieve(this, no_sync)
     if (.not. this%east_boundary) call this%retrieve_east_halo
     if (.not. this%west_boundary) call this%retrieve_west_halo
   end subroutine
+  
+  module subroutine exchange_v(this)
```

Comment:
@d-reynolds

Reply:
Yup, those are the only times that exchange_u/v are used. I thought that exchange_u/v are needed since they have some particular functionality that would be difficult to parameterize within exchange. For example, exchanging over a wider halo_size in one direction (x vs. y).